packages/grafana-perf-tests/run.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
I'm not very familiar with k6 or the object store project but what is the benefit of bash instead of using an [npm bin](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#bin) executable? Feels odd to publish an npm package that has a shell executable.

Reply:
ideally we would require _just_ the k6 binary to be present to run the tests - without node/npm/yarn. I'll have to remove the `yarn build/prepare` calls from the shell script to achieve that, but, once I do, the current approach should work even on windows via gitbash or similar.

Reply:
So an alternative would be just to write this script in typescript instead of bash. I guess that is what @jackw is saying.

Reply:
Yeah I was thinking if the shell script was TS and built with the other files you could eventually do something like `npx @grafana/perf-tests --test="nameOfTest"`.

Reply:
With the bash script, all you need is the k6 binary and the following command - 

```
tar -xzf https://registry.npmjs.org/@grafana/perf-tests/-/tests-9.4.0.tgz -C ./perf-tests
     && k6 run ./perf-tests/object-store-test.js`
```

or 


```
tar -xzf https://registry.npmjs.org/@grafana/perf-tests/-/tests-9.4.0.tgz -C ./perf-tests
     && ./perf-tests/run.sh`
```

this seems pretty rad